optuna_dashboard/ts/components/ModelViewer.tsx

```diff
@@ -0,0 +1,109 @@
+import * as THREE from "three"
+import React, { useEffect, useState } from "react"
+import { Canvas } from "@react-three/fiber"
+import { GizmoHelper, GizmoViewport, OrbitControls } from "@react-three/drei"
+import { STLLoader } from "three/examples/jsm/loaders/STLLoader"
+import { Rhino3dmLoader } from "three/examples/jsm/loaders/3DMLoader"
+import { PerspectiveCamera } from "three"
+
+interface ModelViewerProps {
+  src: string
+  width: string
+  height: string
+  hasGizmo: boolean
+  filetype: string | undefined
+}
+
+function CustomGizmoHelper(): JSX.Element {
```

Comment:
nit: In Optuna Dashboard, basically we use an arrow function syntax to declare React components.

```suggestion
const CustomGizmoHelper: React.FC<{}> = (props) => {
```